### PR TITLE
Updates for 0.34.0

### PR DESCRIPTION
**Pull request by GeopJr**
- Switch to Log module
- Domain change for discordbots.org
- Some typos

### CODE REVIEW

File: src/bot_list.cr [hunk 1]
```diff
@@ -66,28 +65,21 @@ module BotList
...
+      Log.error { "#{response.body? ? response.body : "(No body)"}" }
     end
 
     # Logging for a general exception raised from a provider's request
     private def log_error(provider : Provider, ex : Exception)
-      @logger.error(<<-DOC)
-      Failed to post stats to provider #{provider.name.inspect} (#{ex.class}, #{ex.message})
-      #{ex.backtrace}
-      DOC
+      Log.error(exception: ex) { "Failed to post stats to provider #{provider.name.inspect} (#{ex.class}, #{ex.message})" }
```
Comment (z64) [inline src/bot_list.cr:81]: `exception: ex` will take care of printing the exception class and message

Example: https://carc.in/#/r/980c

File: src/bot_list.cr [hunk 2]
```diff
@@ -66,28 +65,21 @@ module BotList
...
+      Log.error { ex.backtrace }
```
Comment (z64) [inline src/bot_list.cr:82]: For this, I prefer `inspect_with_backtrace`.
Compare these three examples: https://carc.in/#/r/980g

Also worth noting, in the next release (0.35), the exception formatter for `Log.error(exception: ex)` should also print backtraces, so this should be unnecessary eventually.

File: src/bot_list.cr (hunk 1 above)
Comment (GeopJr) [reply src/bot_list.cr]: Should be resolved as of 3de20b7c547fd243d812b6b890636ac3a2aa3b9b
Thanks for the tip!

File: src/bot_list.cr (hunk 2 above)
Comment (GeopJr) [reply src/bot_list.cr]: Should be resolved as of 3de20b7c547fd243d812b6b890636ac3a2aa3b9b
Thanks for the tips!

**Pull request merged**